test/mailers/previews/user_notifier_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_notifier_mailer
+class UserNotifierMailerPreview < ActionMailer::Preview
```

Comment:
Identation Fixed.
This code is for testing and design purposes only. We can preview the email to be sent, without actually send it, just by following this link: http://localhost:3000/rails/mailers/user_notifier_mailer. I used it to make sure the code was correct and that it was generating the email. We can delete it now.